mission-control/docs/topology/lookups/kubernetes.md

```diff
@@ -6,7 +6,7 @@ title: Kubernetes
 
 The Kubernetes component lookup fetches kubernetes resources to be used as components.
 
-```yaml title="kube-check.yml"
+```yaml title="kube-configmap-components.yml"
```

Comment:
Can we use fixtures for all the examples ?

Reply:
https://github.com/flanksource/canary-checker/pull/2212